docs/book/rest-api.md

```diff
@@ -1690,12 +1690,15 @@ Content-Type: application/json
 - **pretty** - If parameter is `true`, response will formatted for humans.
 - **explain** - Return query explanation in addition to result. Values: **full**, **truth**.
 - **metrics** - Return query performance metrics in addition to result. See [Performance Metrics](#performance-metrics) for more detail.
+- **watch** - Set a watch on the query. The parameter must be an integer that will be used as a unique ID within the server. A positive integer indicates to start a watch, a negative integer indicates to end the watch with the absolute value of the integer as its ID. If ending a watch, any supplied queries will be ignored, so the `q` parameter may be omitted in this case. See [Watches](#watches) for more detail.
```

Comment:
re: first comment, can do. It's currently not required to end the watch, but it's merely a supplied option.

re: second comment, it should already? just watch a query like `x = data`?